### PR TITLE
remove pr and issue templates

### PR DESCRIPTION
**Pull request by wxtim**
These changes partially address https://github.com/cylc/cylc-admin/issues/127

Companion of https://github.com/cylc/cylc-flow/pull/5365 and https://github.com/cylc/cylc-uiserver/pull/428 and https://github.com/cylc/cylc-ui/pull/1252

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,12 +40,23 @@ the Contributor Licence Agreement and Certificate of Origin detailed
...
+ - Ash Newport
+ - Garrett Walker
+ - Edgardo Ramírez
  - Mel Hall
- - Bruno Kinoshita
- - Hilary Oliver
+ - Ronnie Dutta
+ - Carol Barno
+ - Giuliano Serrao
+ - Aaron Cole
```
Comment (oliver-sanders) [inline CONTRIBUTING.md:59]: Please don't update this list.

Comment (wxtim) [reply CONTRIBUTING.md:59]: Sorry, My mistake here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,12 +40,23 @@ the Contributor Licence Agreement and Certificate of Origin detailed
 below.
 
 <!-- start-shortlog -->
+ - Hilary Oliver
+ - Matt Shin
```
Comment (wxtim) [inline CONTRIBUTING.md:44]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -31,12 +40,23 @@ the Contributor Licence Agreement and Certificate of Origin detailed
...
  - Oliver Sanders
+ - Sadie Bartholomew
+ - Bruno Kinoshita
+ - David Sutherland
+ - Martin Ryan
```
Comment (wxtim) [inline CONTRIBUTING.md:49]:
```suggestion
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -31,12 +40,23 @@ the Contributor Licence Agreement and Certificate of Origin detailed
...
  - Tim Pillinger
- - Ronnie Dutta
+ - Alex Szabó
+ - Ash Newport
+ - Garrett Walker
+ - Edgardo Ramírez
```
Comment (wxtim) [inline CONTRIBUTING.md:54]:
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -31,12 +40,12 @@ the Contributor Licence Agreement and Certificate of Origin detailed
...
 <!-- start-shortlog -->
+ - Hilary Oliver
  - Oliver Sanders
+ - Bruno Kinoshita
  - Tim Pillinger
- - Ronnie Dutta
  - Mel Hall
- - Bruno Kinoshita
- - Hilary Oliver
+ - Ronnie Dutta
```
Comment (wxtim) [inline CONTRIBUTING.md:48]:
```suggestion
 - Oliver Sanders
 - Tim Pillinger
 - Ronnie Dutta
 - Mel Hall
 - Bruno Kinoshita
```

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**